Hymns/Home/LaunchRouterView.swift

```diff
@@ -0,0 +1,26 @@
+import SwiftUI
+
+struct LaunchRouterView: View {
+    @State var showSplash: Bool = false
+
+    var body: some View {
+        Group { () -> AnyView in
+            if showSplash {
+                return SplashScreenView()
+                    .opacity(showSplash ? 1 : 0)
+                    .onAppear {
+                        DispatchQueue.main.asyncAfter(deadline: .now() + 2.7) {
+                            self.showSplash = false
+                        }
+                }.eraseToAnyView()
+            } else {
+                return HomeContainerView().eraseToAnyView()
+            }
+        }.onAppear {
+            if !UserDefaults.standard.bool(forKey: "didLaunchBefore") {
+                UserDefaults.standard.set(true, forKey: "didLaunchBefore")
+                self.showSplash = true
+            }
+        }
```

Comment:
Can you use the `UserDefaultsManager` to do this? I created a property wrapper so you can probably just use that instead of doing this in onAppear: https://swiftsenpai.com/swift/create-the-perfect-userdefaults-wrapper-using-property-wrapper/

Reply:
Yes, I did this. Can you just check that I did it right. The changes are on my new Lottie PR